public/components/ctas/8.html

```diff
@@ -0,0 +1,11 @@
+<div class="bg-sky-50">
+  <div class="bg-sky-50 p-8 max-w-7xl m-auto lg:flex lg:items-center lg:justify-between sm:p-12">
+    <div>
+      <p class="text-3xl font-bold sm:text-4xl">Ready to grow? <br>
+      <span class="text-blue-700">Start your free trial now.</p>
+    </div>
+    <div class="mt-12 lg:mt-0">
+      <a href="#" class="py-4 px-6 text-xl text-slate-50 bg-blue-700 rounded-xl hover:bg-blue-600">Get Started</a>
+    </div>
+  </div>	
+</div>
```

Comment:
I like the design of this, but the code needs some work.

There's a lot of extra `<div>` elements and quite a few Tailwind CSS classes that aren't needed. Also, there's broken HTML.